vllm_detector_adapter/generative_detectors/base.py

```diff
@@ -66,16 +70,16 @@ def load_template(self, template_path: Optional[Union[Path, str]]) -> str:
         logger.info("Using supplied template:\n%s", resolved_template)
         return self.jinja_env.from_string(resolved_template)
 
-    def apply_task_template(
+    def apply_task_template_to_chat(
```

Comment:
nit: can we just group them a bit with `####` commends or something that way its easier to know which all functions apply for each of the detector type